plugins/right-sizer/src/controller/controller.go

```diff
@@ -122,6 +132,12 @@ func NewController(stop chan struct{}, kubeClientResources util.KubeClientResour
 		o(cfg)
 	}
 
+	missingAllowedNamespaces := funk.RightJoinString(cfg.allowedNamespaces, cfg.allowedUpdateNamespaces)
+	if len(cfg.allowedNamespaces) > 0 && len(missingAllowedNamespaces) > 0 {
```

Comment:
Should we be checking if len > 0 here? I think we'd always want to do this...

Reply:
@rbren If no global namespaces were specified, and we always added any update namespaces to the global list, we would unintentionally limit namespaces for which action-items happen. By default an empty global namespaces list means action-items happen everywhere. I want to make sure this logic makes sense and I'm not missing your point, before I merge.

Reply:
Robert approved this point in [this Slack thread](https://fairwindsops.slack.com/archives/CNQBGRGFN/p1642113005253000?thread_ts=1642112819.251000&cid=CNQBGRGFN)